File: python/Program/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    def __init__(self, id=0, name="", stock=0, price=0.0, jenis="", bahan="", warna=""):
        super().__init__(id, name, stock, price)
        self._jenis = jenis
        self._bahan = bahan
        self._warna = warna
    
    def set_jenis(self, jenis): self._jenis = jenis
    def get_jenis(self): return self._jenis
    
    def set_bahan(self, bahan): self._bahan = bahan
    def get_bahan(self): return self._bahan
    
    def set_warna(self, warna): self._warna = warna
    def get_warna(self): return self._warna

File: java/Program/Aksesoris.java
public class Aksesoris extends Petshop {
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris() {
        super();
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    public Aksesoris(int id, String name, int stock, double price, String jenis, String bahan, String warna) {
        super(id, name, stock, price);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public void setJenis(String jenis) { this.jenis = jenis; }
    public void setBahan(String bahan) { this.bahan = bahan; }
    public void setWarna(String warna) { this.warna = warna; }

    public String getJenis() { return this.jenis; }
    public String getBahan() { return this.bahan; }
    public String getWarna() { return this.warna; }
}